_clubs/natural-areas-association-of-klahanie-naak.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Natural Areas Association of Klahanie (NAAK)

```

Comment:
The title and photo are set up like the Tennis page. Both look good but are not consistent with the layout used in the rest of the site.

Reply:
We don't have banners for all the pages, but for the ones we do, the banner statement (example on line 11 in this file) causes us to render the banner instead of the plain title heading. We can change that behavior globally, but I was trying to up the design factor on some of the pages where we've got nice photos to show off.